packages/@aws-cdk/aws-lambda-event-sources/test/test.kafka.ts

```diff
@@ -0,0 +1,157 @@
+import { expect, haveResource } from '@aws-cdk/assert';
+import * as lambda from '@aws-cdk/aws-lambda';
+import { Secret } from '@aws-cdk/aws-secretsmanager';
+import * as cdk from '@aws-cdk/core';
+import { Test } from 'nodeunit';
+import * as sources from '../lib';
+import { TestFunction } from './test-function';
+
+/* eslint-disable quote-props */
```

Comment:
drop this and remove all the quotes. eslint autofix should do this for you automatically.